rotkehlchen/accounting/export/csv.py

```diff
@@ -138,19 +138,30 @@ def _add_pnl_type(
         dict_event[f'pnl_{name}'] = equation
         cost_basis = ''
         if event.cost_basis is not None:
-            for acquisition in event.cost_basis.matched_acquisitions:
-                if name == 'taxable' and acquisition.taxable is False:
-                    continue
-                if name == 'free' and acquisition.taxable is True:
-                    continue
-
-                index = acquisition.event.index + CSV_INDEX_OFFSET
-                if cost_basis == '':
-                    cost_basis = '='
+            if self.settings.cost_basis_method == CostBasisMethod.ACB:
+                # ACB doesn't have matched acquisitions so we use pure numbers.
+                if name == 'taxable':
```

Comment:
This can be shortened

```python
cost_basis = event.cost_basis.taxable_bought_cost if name == 'taxable' else event.cost_basis.taxfree_bought_cost
cost_basis = str(cost_basis) if cost_basis != 0 else '' 
```